docs/elements/mysql/tips.md

```diff
@@ -10,11 +10,15 @@ parent: Back to Element Guides
 order: 50
 ---
 
-## Tips
+# Tips
 
-### Why can't I connect my database?
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
 
-Check your config file to make sure the port and bind-address have been set.
+{% include callout.html content="<strong>On this page</strong></br><a href=#general>General</a></br><a href=#events>Events</a></br><a href=#events>Events</a></br><a href=#documentation>Documentation</a></br>" type="info" %}
+
+## General
```

Comment:
I'd change this to Authentication or Connection because the tip after this about spaces in column names get lost, and then add a heading to it like "Spaces in Column Names"